### ISSUE TITLE
AXE Violation: aria-describedby (should not be present unless tooltip is active)

### ISSUE THREAD

**Issue by carmanchris31** [valor-software/ng2-bootstrap#5089]
### Bug description or feature request:

`aria-describedby` attribute should only be present when tooltip is shown, otherwise the attribute is considered invalid since the referenced id doesn't exist on the page (aXe violation):
![screen shot 2019-03-04 at 11 01 58 am](https://user-images.githubusercontent.com/9935648/53756485-1881df80-3e6e-11e9-92b3-4a8b7023ec1b.png)

On the [Bootstrap doc/demo page](https://getbootstrap.com/docs/4.0/components/tooltips/#examples) you can see the expected behavior:
![bootstrap-tooltip](https://user-images.githubusercontent.com/9935648/53756472-0ef87780-3e6e-11e9-937e-5877045d1f06.gif)

We should hopefully just be able to conditionalize the attribute by whether the tooltip is shown, something like this:
```js
@HostBinding('attr.aria-describedby') ariaDescribedby = this._tooltip.isShown ? `tooltip-${this.tooltipId}` : null;
```

Original: https://github.com/valor-software/ngx-bootstrap/blob/c9adab66e85aaf93548eda473f924812263716a6/src/tooltip/tooltip.directive.ts#L197

### Plunker/StackBlitz that reproduces the issue:
https://stackblitz.com/edit/ngx-bootstrap-vanaus?file=app%2Fapp.component.html

(aXe extension for reference: https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd)

### Versions of ngx-bootstrap, Angular, and Bootstrap:

ngx-bootstrap:
3.3.0 (since v3.1.4 [commit](https://github.com/valor-software/ngx-bootstrap/commit/583068afb5ce7791aaa17b41dd93813c80ef6615#diff-0be56b3762499fef3173397bad514ec5R193))

Angular:
7 (but not dependent on angular version)

Bootstrap:
n/a

**Comment by Totati** [comment]
I came here facing the same issue. The material team chose to leave a copy of the tooltip's content in the DOM, referring to that id.

**Comment by shainanigans** [comment]
Same issue here as well. @ludmilanesvitiy , I see this had an enhancement label added to it back in March. Has any progress been made on this?
